llvm/test/tools/llvm-objcopy/ELF/srec-writer.test

```diff
@@ -44,64 +48,65 @@ FileHeader:
   Machine:         EM_X86_64
 Sections:
   - Name:            .data1
-# Records for this section should come last.
+## Records for this section should come last.
     Type:            SHT_PROGBITS
     Flags:           [ SHF_ALLOC ]
     Content:         "11111111111111111111"
     Address:         0xEFFFFF
   - Name:            .data2
-# This section overlaps 24-bit address boundary, so we expect
-# its record type to be S3.
+## This section overlaps 24-bit address boundary, so we expect
+## its record type to be S3.
     Type:            SHT_PROGBITS
     Flags:           [ SHF_ALLOC ]
     Content:         "3031323334353637383940"
     Address:         0xFFFFF8
-# Sign-extended addresses are OK.
+## Sign-extended addresses are OK.
   - Name:            .data3
     Type:            SHT_PROGBITS
     Flags:           [ SHF_ALLOC ]
     Address:         0xFFFFFFFF80001000
     Content:         "0001020304"
   - Name:            .text
-# This section's contents exceeds default line length of 16 bytes
-# so we expect two lines created for it. Records for this section
-# should appear before records for the previous section.
+## This section's contents exceeds default line length of 16 bytes
```

Comment:
Nit
```suggestion
## This section's contents exceed default line length of 16 bytes
```
Nit: "content exceeds" or "contents exceed" to be grammatically correct.

Reply:
Done